contrib/examples/generate-blueprint-docs-simple/Kptfile

```diff
@@ -0,0 +1,26 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: bucket
+  annotations:
+    blueprints.cloud.google.com/title: Google Cloud Storage Bucket blueprint
```

Comment:
@bharathkkb  I think there's actually `title` under `info` (ref https://raw.githubusercontent.com/GoogleContainerTools/kpt/main/site/reference/schema/kptfile/kptfile.yaml). Should we use that instead?

Reply:
@morgante Were you referring to [this](https://github.com/GoogleContainerTools/kpt/blob/9e3f7f8e261587b5e19ceade9ae6a1cabe1108fc/site/reference/schema/kptfile/kptfile.yaml#L204)? I think thats the OpenAPI [schema title](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#properties). Looking at the [package info struct](https://github.com/GoogleContainerTools/kpt/blob/9e3f7f8e261587b5e19ceade9ae6a1cabe1108fc/pkg/api/kptfile/v1/types.go#L174-L195) I didnt see title.

Reply:
Good point, yeah we can continue to use this annotation. Resolved.